apps/electron/src/app/commands/Login.command.ts

```diff
@@ -0,0 +1,77 @@
+import { net } from "electron";
+import AppState from "../AppState";
+import EventBus from "../events/EventBus";
+import { Events } from "../events/Events.enum";
+import { Command } from "./CommandManager";
+
+export class LoginCommand implements Command {
+
+  private serverStarted: Promise<void>
+
+  constructor() {
+    this.serverStarted = new Promise((resolve) => {
+      AppState.stateChanged((state) => {
+        state.serverUp ? resolve() : void 0;
```

Comment:
We need this only once, so by default we can unsubscribe directly. This will not affect the promise on top since this one is resolved forever after that. And current state after we logout from app which runs after that inside angular we will never need the LoginCommand anymore.

So we can remove it from Command Manager for now.